css/qfield.css

```diff
@@ -94,8 +94,10 @@ span.shadow-background {
 
 .navbar-brand img {
   display:inline-block;
-  height: 70px;
-  width: 70px;
+  width: auto;
+  height: 52px;
+  margin-top: 8px;
+  max-width: calc(100vw - 100px);
```

Comment:
Very qml-ish solution: ) but it works.

Reply:
I like the vw vh CSS units, they are a great addition :)